catalog/gke/cluster/cluster.yaml

```diff
@@ -37,10 +37,10 @@ spec:
     # https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/gce-pd-csi-driver
     gcePersistentDiskCsiDriverConfig:
       enabled: true
-    # Enable Network Policy by default, for increased security.
+    # Network Policy does not need to be enabled explicitly when using Dataplane V2, it's always on
     # https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy
     networkPolicyConfig:
-      disabled: false
+      disabled: true
```

Comment:
If default is true and not necessary for Dataplane V2, we could remove this

Reply:
removed